llvm/test/tools/llvm-ctxprof-util/Inputs/valid.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "Guid": 1000,
+    "Counters": [1, 2, 3],
+    "Callsites": [
+      [],
+      [
+        {
+          "Guid": 2000,
+          "Counters": [4, 5]
+        },
+        {
+          "Guid": 18446744073709551613,
+          "Counters": [6, 7, 8]
+        }
+    ], 
```

Comment:
Nit, can you align the matching brackets?

Reply:
fixed - the reference file is indented.

Reply:
oh you meant the json not the xml output. sorry. Misread.

Reply:
used a formatter this time.